rosco-web/config/rosco.yml

```diff
@@ -5,7 +5,9 @@ rosco:
   configDir: /some/path/to/config/packer
   jobs:
     local:
-      timeoutMinutes: 20
+      timeoutMinutes: 30
```

Comment:
@rguthriemsft  When baking Windows images, we are using WinRM to connect to and provision the instance. For the Azure Resource Manager Builder, packer automatically creates the WinRM certificate for Windows machines. The object_id, which is the object id of the service principal, is required in order to protect the certificate. Please see the [Azure ARM](https://www.packer.io/docs/builders/azure-arm.html) docs which specify that this is a required parameter when the os_type is 'Windows'.